foundation/foundation-drift-detection/foundation-drift-detection-engine/src/test/java/com/boozallen/drift/detection/CommonCdiSteps.java

```diff
@@ -10,20 +10,23 @@
  * #L%
  */
 
-import com.boozallen.drift.detection.cdi.TestCdiContainer;
+import com.boozallen.aissemble.alerting.core.cdi.AlertingCdiContext;
+import com.boozallen.aissemble.core.cdi.CdiContainer;
 import com.boozallen.drift.detection.cdi.TestCdiContext;
 import com.boozallen.drift.detection.consumer.TestConsumer;
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import org.jboss.weld.environment.se.WeldContainer;
 
+import java.util.List;
+
 public class CommonCdiSteps {
 
     private WeldContainer container;
 
     @Before("@cdi")
     public void setUp() {
-        container = TestCdiContainer.create(new TestCdiContext());
+        container = CdiContainer.create(List.of(new TestCdiContext(), new AlertingCdiContext()));
```

Comment:
I: Allows us to greatly simplify `TestCdiContext` and remove `TestCdiContainer` entirely.